packages/opentelemetry-metrics/test/export/aggregators/Histogram.test.ts

```diff
@@ -40,18 +40,9 @@ describe('HistogramAggregator', () => {
   });
 
   describe('.update()', () => {
-    it('should not update checkpoint', () => {
```

Comment:
why is this not needed ?

Reply:
Aggregators are expected to update the current checkpoint on each `aggregator.update` call. This case asserts the original HistogramAggregator update the checkpoint on `histogramAggregator.reset` -- this is not compliant with the spec and not aligned with other existing aggregators.